contracts/WildcardSteward_v2.sol

```diff
@@ -123,9 +127,10 @@ contract WildcardSteward_v2 is Initializable {
     ) public onlyAdmin {
         assert(tokens.length == _benefactors.length);
         for (uint8 i = 0; i < tokens.length; ++i) {
+            // This index is starting from 0? How do we ensure non conflicting index to already minted tokens?
```

Comment:
Question for the future. As you said, we might need to separate the minting out into a separate smart contract?

Reply:
We would have to write some code to do this in the migration.

Maybe it also makes sense to make the logic controlling minting be in a separate contract. Then it is easy, there is only one contract that controls minting, and it is easier to change that, since it is separate and independent.